inspire_dojson/hep/rules/bd0xx.py

```diff
@@ -262,6 +267,7 @@ def _is_scheme_spires(id_, schema):
         return schema == 'SPIRES'
 
     result_035 = self.get('035', [])
+    id_dict = self.get('id_dict', defaultdict(list))
```

Comment:
I think that this is what has made the external_system_identifiers not deterministic anymore (now the order in them is changes). Not sure if we would want to avoid that.

Reply:
order is irrelevant among different types of identifiers.